Description: <quilt-format introduced>
 .
 iptables-converter (0.9.8-1) unstable; urgency=low
 .
   *  debian format changed to quilt
   *  split master as uplink from debian/sid branch
   *  german words replaced by english terms (Closes: #780688)
Author: Johannes Hubertz <johannes@hubertz.de>
Bug-Debian: https://bugs.debian.org/780688

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- iptables-converter-0.9.8.orig/Makefile
+++ iptables-converter-0.9.8/Makefile
@@ -6,7 +6,7 @@ doc:
 
 clean:
 	@python setup.py clean --bdist-base build
-	@rm -rf *~ *.pyc .coverage build docs/build/* .tox
+	@rm -rf *~ *.pyc .pybuild/ .coverage build docs/build/* .tox
 	@rm -rf iptables_converter.egg-info
 	@rm -rf dist __pycache__ *.py3
 	@(cd docs; make clean)
